Updated ICP and persona in handbook (#17820)
PostHog/posthog.com: contents/handbook/who-we-build-for.md

showTitle: true
---

We define who we build for as ICP, i.e. the team, and the persona, i.e. the actual people using PostHog.

## Our current ICP

AKA our [ideal customer profile](/newsletter/ideal-customer-profile-framework).

> We build for the people building products in **AI-first software teams at any scale**. 

Reviewer: Does this mean that they are building an AI product, or that they work in an AI-first way?

Reading farther down, it's the latter. Maybe...

```suggestion
We build for the people building products in **AI-powered software teams at any scale**. 
```
?

Reviewer 2: we mean that they work in an AI-first way, ie. they use ADEs etc. So I think the original is probably more appropriate?

Reviewer: AI-pilled?

Reviewer 2: I feel like nowadays that's being used both positively and negatively. Is there something about AI-first that doesn't work for you? 

> We build for the people building products in **software teams adopting AI tools enthusiastically at any scale**

is basically what we're getting at, but I think that's too wordy

Reviewer: yeah AI-first just makes me wonder if it means they are building an AI product. Which there are plenty of teams at PostHog for instance that are building services for eg feature flags, and that's not an AI product. But they are AI pilled and would be great users/customers for PostHog.

Reviewer 2: 'AI-powered'?

I don't reeallly think it's confusing tbh - I think it's a lot more likely folks would read that and assume it means they're using AI, not building it, especially in the context of the whole page. (We would have just said 'AI software teams' or 'AI product software teams' otherwise).

Reviewer 3: +1 from me on this being confusing. I think AI-pilled is better, how are you seeing it used negatively?

Reviewer 2: Vibe based on Reddit/HN. A search for anecdata tells me my vibe is fairly weak tho, and if 2 people are now confused I will concede!

> We want to be the first tool that technical founders add to their product.

Reviewer: ```suggestion
We want to be the first tool that technical founders go to when they start building their product.
```

Right? We want them to scaffold it with PostHog code? Or are we aiming for later in their journey, when they have the MVP built?

Reviewer 2: agree!

| &nbsp; | AI-first software teams |
| --- | --- |
| **Description** | Software teams that either have product-market fit and are quickly scaling up with new customers, hiring, and adding more revenue, or are aiming to do this. They believe in an AI-first approach to building and are already familiar with the standard AI tools. |
| **Criteria** | - They have 1 to 500 potential PostHog users, but the company itself could be anything from one employee to thousands<br />- Has lots of revenue or aims to raise or has raised from leading investors<br />- Not hobbyists or cost-conscious bootstrappers, basically|
| **Why they matter?** | - We believe AI-first companies will outperform non-AI-first companies <br />- We're able to efficiently monetize them<br />- Very quick sales cycle<br />- They act as key opinion leaders for earlier-stage startups and slower-moving companies<br />- They have strong opinions on what they need, which helps us build a better product |
| **Examples** | PostHog any time from during YC to IPO, Supabase, ElevenLabs |

## Our current Persona

Persona is the job title or role of the person actually using a product in PostHog. 

> PostHog's initial adoption inside an ICP should come from an _engineering_ persona most of the time, simply because engineers exist first in startups. The more we appeal to engineers at the earliest stages of their startup's life, the more overall growth we will have. 

Reviewer 4: Just to pressure-test this: Why not founders?

To me, the data already suggests that founders are becoming the primary persona of users within PostHog and I feel like more and more of our content is focused on founder personas (e.g. latest video is about when to hire PMs, which is not an engineering decision). 

Or is this a clean edit to accomodate that view:

```suggestion
PostHog's initial adoption inside an ICP should come from an _engineering_ persona most of the time, simply because engineers exist first in startups. This persona can include technical founders. The more we appeal to engineers at the earliest stages of their startup's life, the more overall growth we will have. 
```
